### PR TITLE
Use tabbed amp-selector for description

### LINKED ISSUE #776: Use tabs in Product Sample
I would like to change the Product sample and use tabs instead of the accordion for showing "Description, measurements etc."

@sebastianbenz WDYT?
@ericlindley-g FYI

### PR DESCRIPTION
**Pull request by kul3r4**
Fixes #776 
@sebastianbenz this is the first version

### CODE REVIEW

**Comment by kul3r4** [comment]
@ericlindley-g I deployed it [here](https://amp-by-example-staging.appspot.com/samples_templates/product_page/preview/). WDYT?

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 1]
```diff
@@ -629,15 +637,14 @@ <h2 class="m1">Apple</h2>
     selected
     option="a">ABOUT</div>
   <div role="tabpanel"
-      class="tabContent p1 p">Fruit are righ of vitamins.</div>
+      class="tabContent p1 p">Fruit are rich of vitamins.</div>
```
Comment (ericlindley-g) [inline src/50_Samples_%26_Templates/Product_Page.html]: Grammar edit + lorem ipsum:

"Fruit is rich in vitamins and minerals. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 2]
```diff
@@ -629,15 +637,14 @@ <h2 class="m1">Apple</h2>
...
   <div role="tab"
     class="tabButton h4"
     option="b">SPECS</div>
   <div role="tabpanel"
-    class="tabContent p1 p">Always wash fruits.</div>
+    class="tabContent p1 p">Always wash fruits before eating.</div>
```
Comment (ericlindley-g) [inline src/50_Samples_%26_Templates/Product_Page.html]: Grammar edit + lorem ipsum:

"Always wash fruit before eating. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 3]
```diff
@@ -629,15 +637,14 @@ <h2 class="m1">Apple</h2>
...
   <div role="tab"
       class="tabButton h4"
-      selected
       option="c">MEASUREMENTS</div>
   <div role="tabpanel"
       class="tabContent p1 p">Size may vary.</div>
```
Comment (ericlindley-g) [inline src/50_Samples_%26_Templates/Product_Page.html]: Adding a little lorem ipsum:

"Size may vary. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."

**Comment by ericlindley-g** [comment]
UI looks great, and the text is a good start.

I recommend adding lorem ipsum to pad out the descriptions more so they're at least a paragraph in length. It would be nice to also make at least one tab contain more content, so users can see the tabs vary in size. I think that would make it easier to understand what's happening on the page. WDYT?

**Comment by kul3r4** [comment]
@ericlindley-g Thanks I added the text.
@sebastianbenz PTAL

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 4]
```diff
@@ -235,6 +235,36 @@
...
+    .ampTabContainer {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    .tabButton {
+        list-style: none;
+        flex-grow: 1;
+        text-align: center;
+        cursor: pointer;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: Can you add a bit more padding: `padding: 8px`

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 5]
```diff
@@ -235,6 +235,36 @@
...
+    }
+
+    .tabButton[aria-selected=true] {
+      border-bottom: 5px solid #b60845;
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: s/5px/4px fits better into our ratio scheme (8px, 16px, 32px,...)

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 6]
```diff
@@ -597,28 +627,28 @@ <h2 class="m1">Apple</h2>
...
+  <div role="tab"
+    class="tabButton h4"
+    option="b">SPECS</div>
+  <div role="tabpanel"
+    class="tabContent p1 p">Always wash fruit before eating. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+  <div role="tab"
+      class="tabButton h4"
+      option="c">MEASUREMENTS</div>
+  <div role="tabpanel"
+      class="tabContent p1 p">Size may vary. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</div>
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: s/p1/py1 as the left and right indentation does not fit with the rest of the sample.

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 7]
```diff
@@ -597,28 +627,28 @@ <h2 class="m1">Apple</h2>
...
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Other</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-      </amp-accordion>
-    </div>
+    <!-- ## Tab panels -->
+    <!-- Use `amp-selector` styled as tab panels to add additional data about your product. Learn more about [`amp-selector`](https://www.ampproject.org/docs/reference/components/dynamic/amp-selector). -->
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: Link to our tab panel sample instead. `Learn how to implement tabs in AMP [here](...).

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: src/50_Samples_%26_Templates/Product_Page.html [hunk 8]
```diff
@@ -628,26 +628,27 @@ <h2 class="m1">Apple</h2>
       </form>
 
     <!-- ## Tab panels -->
-    <!-- Use `amp-selector` styled as tab panels to add additional data about your product. Learn more about [`amp-selector`](https://www.ampproject.org/docs/reference/components/dynamic/amp-selector). -->
+    <!-- Use `amp-selector` styled as tab panels to add additional data about your product.
+    Learn how to implement tabs in AMP [here](https://ampbyexample.com/advanced/tab_panels_with_amp-selector/preview/). -->
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product_Page.html]: use relative URLs

**Pull request merged**